doc/api/buffer.md

```diff
@@ -586,7 +586,7 @@ added:
   - v14.18.0
 -->
 
-* Type: {string}
+* {string}
```

Comment:
Why this change?

Reply:
In other cases, we don't prefix types with "Type:", but rather just show the type, for example:
- https://github.com/nodejs/node/blob/main/doc/api/async_context.md?plain=1#L306
- https://github.com/nodejs/node/blob/main/doc/api/buffer.md?plain=1#L1560
- And many others

In my proposed definition (https://github.com/nodejs/api-docs-tooling/issues/348), we also don't prefix types with "Type:".

Reply:
wdyt about adding the missing ones instead? That seems like a useful way to differentiate methods from non-methods.

Reply:
> We don't prefix [far more times](https://github.com/search?q=repo%3Anodejs%2Fnode+%22*+%7B%22+path%3A%2F%5Edoc%5C%2Fapi%5C%2F%2F&type=code) than we do prefix, I don't think it's really "missing" the "Type:".

Well, not an excuse per se xD

Reply:
Got it! I'll change the missing rather than remove the existent